src/test/java/org/wso2/lsp4intellij/utils/FileUtilsTest.java

```diff
@@ -0,0 +1,150 @@
+package org.wso2.lsp4intellij.utils;
+
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.fileEditor.FileDocumentManager;
+import com.intellij.openapi.fileEditor.FileEditor;
+import com.intellij.openapi.fileEditor.FileEditorManager;
+import com.intellij.openapi.fileEditor.TextEditor;
+import com.intellij.openapi.fileEditor.ex.FileEditorManagerEx;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.testFramework.BinaryLightVirtualFile;
+import com.intellij.testFramework.LightVirtualFile;
+import java.io.File;
+import java.net.URISyntaxException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+public class FileUtilsTest {
+
+  @PrepareForTest({FileEditorManagerEx.class, FileUtils.class, Editor.class, File.class, FileEditorManager.class, FileEditor.class, FileEditor[].class})
```

Comment:
remove this

Reply:
Most of these could be removed but there was one that was still needed